objects/src/accounts/storage/slot.rs

```diff
@@ -53,6 +60,15 @@ impl StorageSlotType {
             _ => false,
         }
     }
+
+    /// Returns the empty [Word] for a value of this type.
+    pub fn empty_word(&self) -> Word {
```

Comment:
I would probably rename this into `default_word()`.

We should also update `is_default()` function above to check for storage maps as well.

Reply:
No, I think `is_default()` is only applicable for `StorageSlotType::Value` or `arity 0`. Took me a while, but we need that for serialization:

https://github.com/0xPolygonMiden/miden-base/blob/727916915ddb9a237ffb975a1f7cc2f9f69139dc/objects/src/accounts/storage/mod.rs#L274-L278

If we would filter out `default_maps` we would not know at deserialization of the original value was a `Map` or `Value`